src/type-syncer.ts

```diff
@@ -65,7 +65,14 @@ export function createTypeSyncer(
 
     const syncedFiles: Array<ISyncedFile> = await Promise.all([
       syncFile(filePath, file, syncOpts, dryRun),
-      ...subManifests.map((p) => syncFile(p, null, syncOpts, dryRun)),
+      ...subManifests.map(async (p) =>
+        syncFile(
+          p,
+          await packageJSONService.readPackageFile(p),
```

Comment:
Just curious, what did this change vs just reading the file from the file path in `syncFile`?

Reply:
I honestly don't remember 🤷‍♂️